nflgame/update_players.py

```diff
@@ -104,8 +96,8 @@ def profile_id_from_url(url):
 
 
 def profile_url(gsis_id):
-    resp, content = new_http().request(urls['gsis_profile'] % gsis_id, 'HEAD')
-    if resp['status'] != '301':
+    resp = requests.head(urls['gsis_profile'], params={'id':gsis_id})
+    if resp.status_code != 301:
         return None
     loc = resp['location']
```

Comment:
```suggestion
    loc = resp.headers['location']
```
TypeError: 'Response' object is not subscriptable

Reply:
Fixed this in my py3 branch